src/ctapipe/io/interpolation.py

```diff
@@ -267,25 +269,74 @@ def add_table(self, tel_id, input_table, par_name="pedestal"):
             Telescope id
         input_table : astropy.table.Table
             Table of pointing values, expected columns
-            are ``time`` as ``Time`` column. The calibration
-            parameter column is given through the variable ``par_name``
-        par_name : str
-            Name of the parameter that is to be interpolated
-            ``pedestal`` is used for pedestals, ``gain`` for gain
-            can also be the name of statistical parameters to
-            interpolate the content of StatisticsContainers
+            are ``time`` as ``Time`` column and "gain"
+            for the relative gain data
+        """
+
+        self.required_columns = {"time", "gain"}
```

Comment:
This is wrong here

Reply:
should this be a list?

Reply:
Since the check is an intersection, a set is best.

You can use a `frozenset`